pkg/utils/utils.go

```diff
@@ -155,3 +155,8 @@ func InitTestDB(t *testing.T, fixtureFiles []string) string {
 
 	return dir
 }
+
+// GenerateAVDLink generates a valid AVD Link
+func GenerateAVDLink(cveID string) string {
+	return strings.ToLower(fmt.Sprintf("https://avd.aquasec.com/nvd/%s", cveID))
+}
```

Comment:
So, this should be kind of like `GetPrimaryLink`, `FilterForPrimaryLink`, or something like that.